### ISSUE TITLE
Can you add support for Mon

### ISSUE THREAD

**Issue by talachanmon** [google/myanmar-tools#58]
I have already make the Non-Standard font mostly used Mon.ttf Converter and want to add that tools to google Myanmar-tools, that would help Mon user using non-standard to used Unicode.

Mon to Unicode converter writing in javascript link : [Mon2Uni Converter](https://talachanmon.github.io/achark/)
[Language link ](https://iso639-3.sil.org/code/mnw)

**Comment by sffc** [comment]
Hi!  We'd be happy to add a link to your GitHub project from the Myanmar Tools README file.  Can you upload your source code under an open-source license to GitHub?

**Comment by sven-oly** [comment]
Yes, it would be useful to refer to your code page for Mon developers and users.

And can you point to the Mon font that needs converting into Unicode. It would be very helpful to document what conversions are included in your code.

**Comment by sven-oly** [comment]
I found several non-Unicode Mon fonts including Mon 2012, UniMon / Mon 2010, MonUni, UniMon small, UniMon, Ramanya Unicode, Mon Anonta, and Mon3. Are any of these fonts widely used?

Also, do Mon users know to install GBoard for Mon and other languages of Myanmar on their mobile devices?

**Comment by talachanmon** [comment]
> Hi! We'd be happy to add a link to your GitHub project from the Myanmar Tools README file. Can you upload your source code under an open-source license to GitHub?

You can used freely https://github.com/talachanmon/achark

**Comment by talachanmon** [comment]
> I found several non-Unicode Mon fonts including Mon 2012, UniMon / Mon 2010, MonUni, UniMon small, UniMon, Ramanya Unicode, Mon Anonta, and Mon3. Are any of these fonts widely used?
> 
> Also, do Mon users know to install GBoard for Mon and other languages of Myanmar on their mobile devices?

Mon, Mon 2010, Mon 2011 are non standard font and Mon3Anonat series are unicode that can used My Converter to convert to Unicode. Mostly for Mobile user not used Gboard because of it wrong keyboard layout to get easy typeing Mon, i also need help how to contact Gboard Keyboad who make that keyboard layout we want to change keyboard layout, currently we using Manic Keyboard https://drive.google.com/file/d/147nSr5RS2aMRf76E32_XKc8f06NMU8tn/view?usp=drivesdk
